eq-author-api/schema/tests/commentsConfirmationPage.test.js

```diff
@@ -0,0 +1,242 @@
+const { buildContext } = require("../../tests/utils/contextBuilder");
+
+const { createQuestionnaire } = require("../../utils/datastore");
+
+const {
+  deleteQuestionnaire,
+} = require("../../tests/utils/contextBuilder/questionnaire");
+
+const {
+  queryComments,
+  createComment,
+  deleteComment,
+  updateComment,
+  createReply,
+  updateReply,
+  deleteReply,
+} = require("../../tests/utils/contextBuilder/comments");
+
+describe("comments", () => {
+  let ctx, questionnaire, confirmationId;
+
+  afterEach(async () => {
+    if (!questionnaire) {
+      return;
+    }
+    await deleteQuestionnaire(ctx, questionnaire.id);
+    questionnaire = null;
+  });
+
+  beforeEach(async () => {
+    ctx = await buildContext({
+      metadata: [{}],
+      sections: [
+        {
+          pages: [
+            {
+              confirmation: {},
+            },
+          ],
+        },
+      ],
+    });
+
+    questionnaire = ctx.questionnaire;
+    confirmationId = questionnaire.sections[0].pages[0].confirmation.id;
+  });
+
+  it("An empty comment array is created on new questionnare", async () => {
```

Comment:
Consider this change to make the wording of our tests more consistent

```suggestion
  it("should create an empty comment array on new questionnaires", async () => {
```